doc/Language/objects.pod6

```diff
@@ -4,8 +4,8 @@
 
 =SUBTITLE Object Orientation in Perl 6
 
-Perl 6 is an object oriented language at its core, even though it allows you
-to write programs in other programming styles.
+Perl 6 is an object oriented language at its core, although you can write
```

Comment:
How about this? What do you think.

Perl 6 language is an object oriented language at its core. Although Perl 6 allows programmers to program in multiple paradigms, support for objects is at the heart of the language.